shared/src/models/utils/map-position.ts

```diff
@@ -0,0 +1,18 @@
+import { getCreate } from './get-create';
+import type { MapCoordinates } from './map-coordinates';
+
+export class MapPosition {
+    public readonly type: 'Coordinates';
```

Comment:
I would prefer lowercase strings for this and the other types, as they're more common for string literals.

Keep in mind to change the value in the constructor, too. I think it should also be possible to omit the line in the constructor and do the assignment here. Since you're using `readonly`, you should even be able to omit the type annotation when assigning the value here.

Reply:
All lowercase? I am asking to decide between simulatedRegion or simulated_region or simulatedregion...

Reply:
I chose simulatedRegion, but I am open for the other alternatives